pkg/webhook/util/configuration/configuration.go

```diff
@@ -32,19 +32,14 @@ import (
 	webhookutil "github.com/koordinator-sh/koordinator/pkg/webhook/util"
 )
 
-const (
```

Comment:
This approach works, but it is recommended to consider another modification, eg: make these const values into variable values, and use the `init` function to initialize the values from environment variable values.